x-pack/plugins/ml/public/application/routing/breadcrumbs.ts

```diff
@@ -34,3 +34,10 @@ export const DATA_VISUALIZER_BREADCRUMB: ChromeBreadcrumb = Object.freeze({
   }),
   href: '#/datavisualizer',
 });
+
+export const CREATE_JOB_BREADCRUMB: ChromeBreadcrumb = Object.freeze({
+  text: i18n.translate('xpack.ml.machineLearningBreadcrumbLabel', {
```

Comment:
The ID and text of this breadcrumb are incorrect. the ID should be `xpack.ml.createJobsBreadcrumbLabel` and the text `Create job`
I'm not sure how you got a green build as this ID is identical to the one at the top of the file.